.github/workflows/kernel_tests.yaml

```diff
@@ -34,7 +36,7 @@ jobs:
         working-directory: ./ansible-playbooks
         run: |
           LIBS_V=${{ github.event.inputs.libsversion }}
-          LIBS_VERSION=${LIBS_V:-"master"}
+          LIBS_VERSION=${LIBS_V:-${{ github.ref_name }}}
```

Comment:
Use `ref_name` as default libs version, that is either `master` or the tag name.